frontend/src/pages/App/index.tsx

```diff
@@ -98,85 +101,88 @@ function App(): JSX.Element {
   }
 
   return (
-    <UserContext.Provider value={userClaims}>
-      <BrowserRouter>
-        <Switch>
-          <Route
-            exact
-            path={ROUTES.SIGN_IN}
-            render={() => <SignInPage setClaims={setClaims} />}
-          />
-          <Route exact path={ROUTES.SIGN_UP} render={() => <SignUpPage />} />
-          <Route
-            exact
-            path={ROUTES.EVENT_SIGN_IN}
-            render={() => <EventSignInPage />}
-          />
-          <Route
-            exact
-            path={ROUTES.EVENT_RSVP}
-            render={() => <EventRsvpPage />}
-          />
-          <Route
-            exact
-            path={ROUTES.EVENT_SIGN_IN_OPTIONS}
-            render={() => <EventSignInOptionsPage />}
-          />
-          <Route
-            exact
-            path={ROUTES.HOME}
-            render={props => InducteeRoutingPermission(EventsPage)(props)}
-          />
-          <Route
-            exact
-            path={ROUTES.POINTS}
-            render={props => InducteeRoutingPermission(PointsPage)(props)}
-          />
-          <Route
-            exact
-            path={ROUTES.INDUCTEES}
-            render={props =>
-              OfficerRoutingPermission(InducteePointsPage)(props)
-            }
-          />
-          <Route
-            exact
-            path={ROUTES.CALENDAR}
-            render={props => InducteeRoutingPermission(CalendarPage)(props)}
-          />
-          <Route
-            exact
-            path={ROUTES.INTERVIEW_SCHEDULING}
-            render={props =>
-              InducteeRoutingPermission(InterviewSchedulingPage)(props)
-            }
-          />
-          <Route
-            exact
-            path={ROUTES.EVENTS}
-            render={() => <QueriedEventPage />}
-          />
-          <Route
-            exact
-            path={ROUTES.EVENT_DETAILS}
-            render={props => InducteeRoutingPermission(EventDetailsPage)(props)}
-          />
-          <Route
-            exact
-            path={ROUTES.EVENT_EDIT}
-            render={props => OfficerRoutingPermission(EventEditPage)(props)}
-          />
-          <Route
-            exact
-            path={ROUTES.FORBIDDEN}
-            render={props => InducteeRoutingPermission(ForbiddenPage)(props)}
-          />
-          <Route
-            exact
-            path={ROUTES.NOT_FOUND}
-            render={props => InducteeRoutingPermission(NotFoundPage)(props)}
-          />
-          {/* <Route
+    <QueryClientProvider client={queryClient}>
+      <UserContext.Provider value={userClaims}>
+        <BrowserRouter>
+          <Switch>
+            <Route
+              exact
+              path={ROUTES.SIGN_IN}
+              render={() => <SignInPage setClaims={setClaims} />}
+            />
+            <Route exact path={ROUTES.SIGN_UP} render={() => <SignUpPage />} />
+            <Route
+              exact
+              path={ROUTES.EVENT_SIGN_IN}
```

Comment:
I just merged the qr code branch into master, so you can update the routes to reflect that when merging :)

Reply:
Yeah for sure, thanks!